testing/run_tests.py

```diff
@@ -402,7 +401,7 @@ def make_test(name, flags=None, extra_env=None):
       make_test('embedder_a11y_unittests'),
       make_test('embedder_proctable_unittests'),
       make_test('embedder_unittests'),
-      make_test('fml_unittests', flags=[FML_UNITTESTS_FILTER] + repeat_flags),
+      make_test('fml_unittests', flags= repeat_flags),
```

Comment:
repeat_flags is the default value.

```suggestion
      make_test('fml_unittests'),
```